fetch.bs

```diff
@@ -6710,6 +6748,364 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+
+       <li><p>If <var>otherDocument</var> is not
+       <a>allowed to use</a> the <a>policy-controlled feature</a>
+       {{PermissionPolicy/"deferred-fetch"}}, then return 0.
+
+       <li>
+        <p>Set <var>quota</var> be 640 kibibytes.
+        <p class="note allow-2119">640kb should be enough for everyone.
+
+       <li><p>If <var>otherDocument</var> is <a>allowed to use</a> the
+       <a>policy-controlled feature</a> {{PermissionPolicy/"deferred-fetch-minimal"}}, then
+       decrement <var>quota</var> by <a>deferred-fetch delegated quota</a>.
+      </ol>
+
+     <li>
+      <p>Otherwise:
+
+      <p class=note>This is a subframe, check if it inherits quota from its parent container based
+      on permissions policy. This would only be the case if the subframe and its parent do not share
+      quota, as quota is either shared or delegated.
+
+      <ol>
+       <li><p>Let <var>otherContainer</var> be <var>otherNavigable</var>'s
+       <a>navigable container</a>.
+
+       <li>
+        <p>If <var>otherContainer</var>'s <a>node document</a> and <var>otherDocument</var> do
+        not <a>share deferred-fetch quota</a>, then:
+
+        <ol>
+         <li><p>If <var>otherContainer</var>'s <a>subframe reserved deferred-fetch quota</a> is
+         <a>optional subframe deferred-fetch quota</a>, and <var>otherDocument</var> is
+         <a>allowed to use</a> the <a>policy-controlled feature</a>
+         {{PermissionPolicy/"deferred-fetch"}}, then increment <var>quota</var> by
+         <a>optional subframe deferred-fetch quota</a>.
+
+         <li><p>Otherwise, if <var>otherContainer</var>'s
+         <a>subframe reserved deferred-fetch quota</a> is
+         <a>minimal subframe deferred-fetch quota</a> and <var>otherDocument</var>
+         is <a>allowed to use</a> the <a>policy-controlled feature</a>
+         {{PermissionPolicy/"deferred-fetch-minimal"}}, then increment <var>quota</var> by
+         <a>minimal subframe deferred-fetch quota</a>.
+        </ol>
+      </ol>
+
+     <li>
+      <p><a for=list>For each</a> <a>deferred fetch record</a> <var>deferredRecord</var> of
+      <var>otherDocument</var>'s <a for=fetch>fetch group</a>'s
+      <a for="fetch group">deferred fetch records</a>:</p>
+      <ol>
+       <li><p>Let <var>requestLength</var> be the <a>total request length</a> of
+       <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>.
+
+       <li><p>Decrement <var>quota</var> by <var>requestLength</var>.
+
+       <li><p>If <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>'s
+       <a for=request>URL</a>'s <a for=url>origin</a> is <a>same origin</a> with <var>origin</var>,
+       then decrement <var>quotaForOrigin</var> by <var>requestLength</var>.
+      </ol>
+    </ol>
+
+   <li>
+    <p>Otherwise, if <var>otherNavigable</var> is not a <a for=/>top-level traversable</a>, and its
+    <a for=navigable>parent</a>'s <a>active document</a> and <var>document</var>
```

Comment:
```
 if otherNavigable is not a top-level traversable, and its parent’s active document and document share deferred-fetch quota,
```

Given the following example

```
root document (a.com) -> <div> -> frame_a (a.com) -> <div> -> frame_b (a.com)

```

* What is `parent's active document` when `otherNavigable` is `frame_a`, is it root document?
* What is `parent's active document` when `otherNavigable` is `frame_b`, is it `frame_a` or `frame_a`'s document?

Reply:
1. yes
2. frame_a